requirements.txt

```diff
@@ -10,3 +10,4 @@ OpenEXR
 pytest
 mathutils
 dataclasses
+colorsys
```

Comment:
The `colorsys` package is part of the standard library and cannot be installed via pip. (This is why the tests fail)

```suggestion

```